### PR TITLE
Brush width

### PR DESCRIPTION
**Pull request by rbischof**
This PR adds the functionality that users can press on the volume buttons during the drawing phase to increase or decrease the brush size.

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java [hunk 1]
```diff
@@ -138,6 +140,17 @@ public void setCircleRadius(int circleRadius) {
...
     }
 
+    public void setDrawWidth(int newWidth) {
+        drawWidth = newWidth;
+        for(Paint paint : colors) {
+            paint.setStrokeWidth(drawWidth);
+        }
+    }
+
+    public int getDrawWidth() {
```
Comment (Gondolav) [inline app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java]: Here you can add VisibleForTesting

File: app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java [hunk 2]
```diff
@@ -138,6 +140,17 @@ public void setCircleRadius(int circleRadius) {
         this.circleRadius = circleRadius;
     }
 
+    public void setDrawWidth(int newWidth) {
```
Comment (Gondolav) [inline app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java]: This one can be package-private

File: app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java (hunk 1 above)
Comment (rbischof) [reply app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java]: No, this is used in DrawingActivity when a button is pressed.

File: app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java (hunk 2 above)
Comment (rbischof) [reply app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java]: You are right, thanks.

File: app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java [hunk 3]
```diff
@@ -146,6 +147,21 @@ public void setCircleRadius(int circleRadius) {
...
+     * @param newWidth the new width of the brush
+     */
+    protected void setDrawWidth(int newWidth) {
+        drawWidth = newWidth;
+        for(Paint paint : colors) {
+            paint.setStrokeWidth(drawWidth);
+        }
+    }
+
+    protected int getDrawWidth() {
```
[the remark this answers is not in the document]
Comment (rbischof) [reply app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java]: But protected ensures, that subclasses will see this method, right? 
What if we were to implement a subclass of PaintView in the future?

[the remark this answers is not in the document]
Comment (Gondolav) [reply app/src/main/java/ch/epfl/sweng/SDP/game/drawing/PaintView.java]: Ok I see your point

**Pull request merged**